Makefile

```
@@ -9,11 +9,13 @@ SHELL := env PATH=$(GO_BIN):$(PATH) $(SHELL)
 .PHONY: format
 format:
 	@echo "Formatting..."
+	@cue fmt internal/cueutils/**/**/*.cue internal/cueutils/**/*.cue
```

Would this suffice to match all the cue files?

```suggestion
	@cue fmt internal/cueutils/**/*.cue
```

Same applies to the other `cue fmt` command below.